andela/wg-millennium-falcon
# #157731574 upgrade nutrition diary

### What does this PR do?

It enables a  user to include food he/she actually ate during the day in addition to what was planed with the nutritional plan.

### How should this be manually tested?
1. create a user account and use same account and login 
2. Under the menu select nutrition 
3. From the nutrition dropdown, menu _select nutrition plan_
4. Click the green buttonbutton that says _add nutrition plan_
5. In the table click the blue button link that says _add new meal item_ and fill in the form that comes right after clicking.
6. After adding the meal, you should be able to see the meal displayed in the nutrition table 
7. To add an extra meal eaten, just hover the mouse over the meal item you want to add the extra meal to and click the pencil over paper icon that appears just besides the meal name and its second icon to right
8. Fill in the field that says _Extra meal_
9. Save and see your changes applied immediately to the meal in the nutrition table on the page

<img width="1440" alt="add_meal" src="https://user-images.githubusercontent.com/32771081/41093449-0393e2a6-6a54-11e8-9671-fb6bae4361bd.png">

<img width="1440" alt="edit_meal" src="https://user-images.githubusercontent.com/32771081/41093517-3600edd8-6a54-11e8-9d9b-2bad53efb6de.png">

Realted story: [#157731574](https://www.pivotaltracker.com/story/show/157731574)
